app/src/main/java/pl/skipcode/basekotlinapp/feature/main/fragments/dashboard/ui/DashboardFragment.kt

```diff
@@ -41,12 +41,19 @@ class DashboardFragment : DashboardContract.View, BaseFragment(){
         btnLogout.setOnClickListener{
             clickLogout()
         }
+        tvUsername.setOnClickListener{
+            clickUsername()
+        }
     }
 
     override fun clickLogout() {
         presenter.logoutUser()
     }
 
+    override fun clickUsername() {
```

Comment:
wywaliłbym te jednolinijkowe metody clickLogout() clickUsername() itd i po prostu w onClicku wywolywalbym presenter.costam(), to prawdopodobnie zrobiles na potrzeby testow, ale imho to nadmiarowe i zaciemnia kod